.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+# Configuring dependabot.yml: https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
```

Comment:
@chern Is this really the root directory? Not .github/workflows?

Reply:
@rlorenzo Dependabot will check inside .github/workflows when the directory is set as "/" (https://docs.github.com/en/github/administering-a-repository/keeping-your-actions-up-to-date-with-github-dependabot#enabling-github-dependabot-version-updates-for-actions)